kops

```diff
@@ -298,7 +315,7 @@ def _cmd_create(args):
         _run_kops([
             'create', 'cluster', '--zones', zones, '--master-zones', zones,
             '--topology', 'private', '--networking', 'flannel',
-            '--master-size', 'm4.large', '--node-size', 'm4.large',
+            '--master-size', 't2.medium', '--node-size', 't2.medium',
```

Comment:
Why do you change these values? Again I think they should be user configurable though instead of hardwired. I would make these values parameterized in another PR.